make_count/make_count.tex

```diff
@@ -0,0 +1,293 @@
+
+We now refer to the definition of symmetric distance provided in the \href{https://www.overleaf.com/project/60d214e390b337703d200982}{Proof Definitions} document; the definition is copied here for convenience:
+
+\begin{definition}[Symmetric distance]
+\label{defn:symm-dist}
+Let $u,v$ be vectors of elements drawn from domain $\mathcal{X}$. Define $m_v(\ell)$ as the multiplicity of element $\ell$ in vector $v$. For example, if $v$ contains five instances of the number ``21'', then $m_v(21) = 5$.
+
+A definition of the symmetric distance between $u$ and $v$, then, is $$d_{\text{Sym}}(u,v) = \sum_{z\in \mathcal{X}} |m_u(z) - m_v(z)|.$$
+\end{definition}
+
+\question{How should I refer readers to a definition located in another document? I know how to use \texttt{$\backslash$label\{\ldots\}} and \texttt{$\backslash$ref\{\ldots\}}, but that's only for referring to definitions, sections, etc. located in the same doc.}
+
+\question{As a follow-up to the question above, if there's not a good way to refer to definitions in proofs, should important definitions be copied into the proof doc (as above), or should I remove ``in-proof'' definitions and rely on readers to track down the right definition in the proof definitions document, which is a document that may be continuously updated for the lifetime of the OpenDP project?}
+
+Combining the assumptions that $\texttt{inf\_cast}(\din, \texttt{TO}) \leq \dout$ and that \texttt{v}, \texttt{w} are $\din$-close under the symmetric distance metric means that
+\begin{equation}
+    d_\text{Sym}(\texttt{u}, \texttt{v})
+    \leq \din \leq \dout.
+\end{equation}
+
+Let $\mathcal{X}$ be the domain of all elements of type \texttt{TIA}. Therefore, we see that the symmetric distance  between \texttt{u} and \texttt{v} is 
+\begin{equation}
+\label{eq:expand-dymm-dist}
+    d_\text{Sym}(\texttt{u}, \texttt{v}) = \sum_{z\in \mathcal{X}} |m_{\texttt{u}}(z) - m_{\texttt{v}}(z)| \leq \din \leq \dout.
+\end{equation}
+
+The $\function$ used in \texttt{MakeCount} sums over a single data type, namely a row. Let \texttt{rows} be a one-element domain, where every element of type \texttt{TIA} is considered to be the same element of \texttt{rows}; and let the single element be called \texttt{row}. Also, as in the \href{https://www.overleaf.com/project/60d215bf90b337ac02200a99}{Pseudocode definitions} document, let \texttt{len(vec)} be a function that returns the number of rows in vector \texttt{vec}.

```

Comment:
The pseudocode_defns doc doesn't use the term "rows" when defining the length of a vector.  I think you can do this more concisely by starting with stating and proving the claim that len(u) = sum_z m_u(z), and then applying triangle inequality to bound |len(u)-len(v)|

Reply:
(So no need to introduce the data type rows.)

Reply:
Done. Let me know if the proof that "len(u) = sum_z h_u(z)" should be more or less detailed (or otherwise changed). I typed a "question" in the tex document to encourage a closer look at this proof.